src/main/java/com/revature/log/IntEvalLogger.java

```diff
@@ -0,0 +1,9 @@
+package com.revature.log;
+
+import org.apache.log4j.Logger;
+
+public class IntEvalLogger {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1118)